src/components/Restoration/BusinessName.vue

```diff
@@ -155,11 +155,12 @@ export default class BusinessName extends Mixins(CommonMixin, DateMixin, NameReq
   /**
    * Fetches and validation a NR.
    * @param nrNum the NR number
+   * @param businessId the business id
    * @param phone the phone number to match
    * @param email the email address to match
    * @returns a promise to return the NR, or throws a printable error
    */
-  async fetchAndValidateNr (nrNum: string, phone: string, email: string): Promise<NameRequestIF> {
+  async fetchAndValidateNr (nrNum: string, businessId: string, phone: string, email: string): Promise<NameRequestIF> {
```

Comment:
https://github.com/kzdev420/bcrs-shared-components/blob/8dc1f0a920dcccabd46b1cfe57eb733131a4f6b7/src/components/correct-name/CorrectNameRequest.vue#L176-L178

At the Shared-component-correctNR, the Fetch&ValidateNR func has 4 params including businessId.
So, I added the businessId Arguments at the Create-UI too